dradis.py

```diff
@@ -965,25 +965,42 @@ def get_attachment(self, project_id: int, node_id: int, filename: int):
 
         return result
 
-    def create_attachment(self, project_id: int, node_id: int):
-        """Create a new attachment
+    def create_attachment(self, project_id: int, node_id: int, attachment: Path, *args: Path):
+        """Create new attachment() on a node
 
         :param project_id: ID for the project
         :param node_id: ID for the node to create attachment for
+        :param attachment: Path of attachment to upload to a node
+        :*args: path(s) of additionnal attachment(s) to add to the upload request (optional)
         """
-
-        raise NotImplementedError()
+        # HTTP REQUEST TYPE
+        req_type = "POST"
 
         # Add required header to set
         self._add_project_header(project_id)
 
-        # endpoint = self._ATTACHMENT.replace("<node_id>", str(node_id))
+        # Remove default application/json content type
+        self.__headers.pop('Content-type',None)
+        
+        # BUILD URL
+        endpoint = self._ATTACHMENT.replace("<node_id>", str(node_id))
+        url = self.__url+endpoint
 
-        # Set the data
-        # data = {}
+        # Add file to upload
+        if attachment.is_file():
```

Comment:
Why not just remove the whole "attachment" argument, since *args already covers the whole functionality? It could  be renamed to *attachments for clarity. It would remove some code duplication for the logic of adding files.